lib/puppetserver/utils/file_utils.rb

```diff
@@ -0,0 +1,64 @@
+require 'fileutils'
+
+module Puppetserver
+  module Utils
+    module FileUtilities
+
+      def self.ensure_dir(setting)
+        if !File.exist?(setting)
+          FileUtils.mkdir_p(setting, mode: 0750)
+          user, group = find_user_and_group
+          FileUtils.chown(user, group, setting)
+        end
+      end
+
+      def self.ensure_file(location, content, mode)
+        if !File.exist?(location)
+          Puppetserver::Utils::FileUtilities.write_file(location, content, mode)
+        end
+      end
+
+      def self.validate_file_paths(one_or_more_paths)
+        return unless one_or_more_paths.any?
```

Comment:
You need to return an empty array here, we expect this method to not return nil.

Reply:
Thanks!  Added that and a spec that should have been there to catch that.